Gemfile

```diff
@@ -85,7 +85,8 @@ gem 'willow_sword', git: 'https://github.com/notch8/willow_sword.git'
 gem 'blacklight', '~> 6.7'
 gem 'blacklight_oai_provider', '~> 6.0'
 
-gem 'hyrax', '~> 2.5.1'
+gem 'hyrax', github: 'samvera/hyrax', branch: 'release-2.8'
```

Comment:
this branch may now be deleted. use `~> 2.8.0` instead?

it may also be okay to use `~> 2.8`, if the various adopters are comfortable with that.